contrib/pod-master/podmaster.go

```diff
@@ -35,14 +35,15 @@ import (
 )
```

Comment:
agree podmaster changes shouldnt be in here. those were a WIP thing i was playing with just to get  a jump start.